src/prpy/planning/retimer.py

```diff
@@ -154,6 +154,15 @@ def __init__(self, do_blend=True, do_shortcut=True, blend_radius=0.5,
             'time_limit': float(timelimit),
         })
 
+    @PlanningMethod
+    def RetimeTrajectory(self, robot, path, options=None, **kw_args):
+        from copy import deepcopy
+        if options is None:
+            options = {}
+        new_options = deepcopy(options)
+        new_options['time_limit'] = kw_args.get('timelimit', 3.)
+        return super(HauserParabolicSmoother, self).RetimeTrajectory(
+            robot, path, options=new_options, **kw_args)
```

Comment:
Sorry, I wasn't clear on my previous comment. Hauser smoother expects the option to be named ```time_limit``` internally. This line allows us to have the ```HauserParabolicSmoother``` use ```timelimit``` and transfer it to the ```time_limit``` option to be passed on to the actual Hauser smoother/planner.